CHANGELOG.md

```diff
@@ -81,6 +81,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - For a given queue, the `sqs-message-consumer` Lambda will now only schedule workflows for rules matching the queue **and the collection information in each queue message (if any)**
     - The consumer also now only reads each queue message **once per Lambda invocation**, whereas previously each message was read **once per queue rule per Lambda invocation**
   - Fixed bug preventing the deletion of multiple SNS rules that share the same SNS topic
+- **CUMULUS-1997**
```

Comment:
I assume this will have to be moved under the heading for "Unreleased" we merge the release back into master

Reply:
yes, trying to get that in and make sure to keep the changelog correct

Reply:
Which will be fun now that I added 2 more sections to the changelog.